bpfd-operator/test/integration/suite_test.go

```diff
@@ -122,10 +122,9 @@ func TestMain(m *testing.M) {
 			clusters.DeleteManifestByYAML(ctx, env.Cluster(), bpfdConfigMap)
 			waitForBpfdConfigDelete(ctx, env)
 			cleanupLog("deleting bpfd namespace")
-			return  env.Cluster().Client().CoreV1().Namespaces().Delete(ctx, internal.BpfdNs, metav1.DeleteOptions{})
+			return env.Cluster().Client().CoreV1().Namespaces().Delete(ctx, internal.BpfdNs, metav1.DeleteOptions{})
```

Comment:
Was this change just gofmt or something?

Reply:
Yes, gofmt